processing/data_collection/gazette/pipelines.py

```diff
@@ -61,3 +62,51 @@ def process_item(self, item, spider):
             if spider.start_date > item.get("date"):
                 raise DropItem("Droping all items before {}".format(spider.start_date))
         return item
+
+
+class DocToPdfPipeline:
+    """
+    Convert a doc[x] file to pdf
+    """
+
+    def process_item(self, item, spider):
+        # if item is not a doc, skip it
+        if not self.is_doc(item["files"][0]["path"]):
+            return item
+        # it's doc[x]. Convert it to pdf
+        doc_path = os.path.join(FILES_STORE, item["files"][0]["path"])
+        # use libreoffice writer to convert
+        command = f"lowriter --convert-to pdf --outdir {FILES_STORE}/full {doc_path}"
+        subprocess.run(command, shell=True, check=True)
+        if doc_path.endswith("doc"):
+            pdf_path = doc_path[:-3] + "pdf"
+        elif doc_path.endswith("docx"):
+            pdf_path = doc_path[:-4] + "pdf"
+        else:
+            pdf_path = doc_path + ".pdf"
+        os.unlink(doc_path)
```

Comment:
Please, don't. We need to keep the original files for accountability. Otherwise, people might argue we _edited_ the file when converting to PDF. The only way to be accountable is to keep the original file we've got from the original source.